test/e2e/app-dir/turbo-resolve-extensions/next.config.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @type {import('next').NextConfig}
+ */
+const nextConfig = {
+  turbo: {
+    resolveExtensions: ['png', 'tsx', 'ts', 'jsx', 'js', 'json'],
+  },
+  webpack(config) {
+    config.resolve.extensions = ['.png', '.tsx', '.ts', '.jsx', '.js', '.json']
+    return config
+  },
```

Comment:
Can we just have a shared field both can read from?

Reply:
yes. It's only a test case

Reply:
no, I meant have a field in next.config.js which both turbopack and webpack read from

Reply:
not in this test, but in general

e.g. `experimental.resolveExtensions` for now and read that in `build/webpack-config.ts` as well

Reply:
I think it would be better to introduce an option that you don't need to add twice